### PR TITLE
add tcp check support

### PR DESCRIPTION
**Pull request by rphillips**
* Includes banners
* Adds a jitter to starting checks to spread them out over 30 seconds

**Todo:**
- [x] Add banner matches

### CODE REVIEW

File: check/check_http.go [hunk 1]
```diff
@@ -8,14 +8,14 @@ import (
 	"crypto/x509"
 	"encoding/json"
 	log "github.com/Sirupsen/logrus"
+	"github.com/racker/rackspace-monitoring-poller/metric"
```
Comment (itzg) [inline check/check_http.go]: Thanks, as you noticed I hadn't run a gofmt on the code after adding those.

File: types/scheduler.go [hunk 2]
```diff
@@ -38,6 +39,9 @@ func (s *Scheduler) Close() {
 }
 
 func (s *Scheduler) runCheck(ch check.Check) {
+	// Spread the checks out over 30 seconds
+	jitter := rand.Intn(30000) + 1
```
Comment (itzg) [inline types/scheduler.go]: Could put `30000` in constants or config

Comment (rphillips) [inline types/scheduler.go]: fixed

File: .circle.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+  environment:
```
Comment (dimtruck) [inline .circle.yml]: i think environment needs to be under machine and post under dependencies keys

Comment (rphillips) [inline .circle.yml]: yeah

Comment (dimtruck) [inline .circle.yml]: Never mind. already fixed!

**Comment by itzg** [comment]
+1

**Pull request merged**